package.json

```diff
@@ -421,5 +421,6 @@
   "volta": {
     "node": "16.15.0",
     "yarn": "1.22.18"
```

Comment:
Surely this needs to be updated or removed.

Reply:
Also, we should be able to remove yarn-deduplicate from `dependencies`

Reply:
Done and done.